Libraries/Text/Text/RCTFontScaleRamp.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import <React/RCTConvert.h>
+
+typedef NS_ENUM(NSInteger, RCTFontScaleRamp) {
```

Comment:
General comment: should we be referring to this as a "Dynamic Type Ramp" instead of a "Font Scale Ramp"? Question applies to the public property name as well.

Reply:
Naming stuff is hard. ¯\_(ツ)_/¯
But yeah, I do like "Dynamic Type Ramp" a lot better.